tests/fixtures/name-matches-to-wikidata.json

```diff
@@ -0,0 +1,96 @@
+{
+  "compareFunction": "name-matches-to-wikidata",
+  "fixtures": [
+    {
+      "description": "Test OSM name different from Wikidata name",
+      "expectedResult": {
+        "result:name_matches_to_wikidata": false
+      },
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "osm:id": 427818536,
+          "osm:type": "way",
+          "name": "Central Park is now something else",
+          "wikidata": "Q160409",
+          "osm:version": 1
+        },
+        "geometry": null
+      },
+      "oldVersion": null
+    },
+    {
+      "description": "Test OSM name matches with aliases on Wikidata",
+      "expectedResult": {},
```

Comment:
> comparators should pass on as much useful knowledge as possible to a human reviewer to make the final decision.

Totally agree @planemad, 💯 The easier bit is returning `"result:name_matches_to_wikidata": true` from the compare function. The harder one is on `osmcha`'s side. Just to keep the scale of things manageable, `osmcha` stores just the features that the comparators have flagged for being potentially harmful/problematic and not all the features. Yes, `osmcha` has all the changesets but not all the features.

I am curious to hear more on this, shall we create a separate ticket for the same?

---

cc: @willemarcel @batpad @geohacker @amishas157